templates/airflow-ec2.j2

```diff
@@ -283,9 +283,15 @@ Resources:
           # Clone Airflow DAGs repository
           git clone https://github.com/Sage-Bionetworks-Workflows/orca-recipes-airflow.git /root/orca-recipes-airflow
 
+          # Get AWS Secret airflow_password and store as variable
+          airflow_password=$(aws secretsmanager get-secret-value --secret-id airflow_password --query SecretString --region us-east-1 | tr -d '"')
```

Comment:
**Nit (_i.e._ [optional](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.)):** You can get rid of the quotes by configuring the AWS CLI with `--output text`. 

```suggestion
          airflow_password=$(aws --output text secretsmanager get-secret-value --secret-id airflow_password --query SecretString --region us-east-1)
```